Sources/XiEditor/EditView.swift

```diff
@@ -604,6 +613,22 @@ final class EditView: NSView, NSTextInputClient, TextPlaneDelegate {
 
                 let x = dataSource.gutterWidth - (gutterXPad + CGFloat(gutterTL.width))
                 let y0 = yOff + dataSource.textMetrics.ascent + linespace * CGFloat(lineIx)
+
+                for annotationType in gutterAnnotations {
+                    var annotations = annotationsForLines[lineIx]![annotationType]!
+                    let annotation = annotations.next()
+
+                    if annotation != nil {
+                        let color = annotationColor(for: annotation!.annotation)
+
+                        if annotationType == AnnotationType.deleted {
+                            renderer.drawSolidRect(x: GLfloat(dataSource.gutterWidth - CGFloat(gutterAnnotationWidth)), y: GLfloat(yOff + dataSource.textMetrics.ascent + linespace * CGFloat(lineIx - 1)), width: GLfloat(gutterAnnotationWidth), height: GLfloat(3.0), argb: color)
```

Comment:
Nit: Maybe you want a specific name for the height of deleted annotations? Something like `deletedAnnotationHeight`